pontoon/settings/base.py

```diff
@@ -476,6 +476,7 @@ def path(*args):
     "STYLESHEETS": PIPELINE_CSS,
     "JAVASCRIPT": PIPELINE_JS,
     "JS_COMPRESSOR": "pipeline.compressors.terser.TerserCompressor",
+    "CSS_COMPRESSOR": "pipeline.compressors.terser.NoopCompressor",
```

Comment:
That one doesn't exist, it's in the parent, https://github.com/jazzband/django-pipeline/blob/master/pipeline/compressors/__init__.py#L256

Weird that that does something constructive on stage